jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,838 @@
+Python-like syntax; LISP-like flexibility; strong C, C++, JS, Python interop;
+and best-in class metaprogramming.
+
+This release includes improvements in the following areas:
+* new language features (user defined literals, private imports, strict effects, `iterable[T]`, dot-like operators, block arguments with optional params)
+* new compiler features (`nim --eval:cmd`, custom nimscript extensions, customizable compiler messages)
+* major improvements to `--gc:arc`, `--gc:orc`
+* correctness and performance of integer and float parsing and rendering in all backends
+* significant improvements in error messages, showing useful context
+* doc generation logic and documentation, in particular `runnableExamples` now works in more contexts and replaces `code-block`.
+* made JS, VM and nimscript backend more consistent with C backend, allowing more modules to work with those backends, including the imports from `std/prelude`; the test suite now standardizes on testing stdlib modules on each major backend (C, JS, VM).
+* support for Apple silicon/M1, 32-bit RISC-V, CROSSOS, improved support for NodeJS backend
+* major improvements to the following modules: `system, math, random, json, jsonutils, os, typetraits, wrapnils, lists, hashes` including performance improvements
+* deprecated a number of error prone or redundant mis-features
+
+
+# Installing Nim 1.6
+We recommend everyone to upgrade to 1.6:
+
+## New users
+
+Check out if the package manager of your OS already ships version 1.6 or
+install it as described [here](https://nim-lang.org/install.html).
+
+
+## Existing users
+
+If you have installed a previous version of Nim using `choosenim`,
+getting Nim 1.6 is as easy as:
+
+```bash
+choosenim update self
+choosenim update stable
+```
+
+If you don't have `choosenim`, you can follow the same
+[install link](https://nim-lang.org/install.html) as above.
+
+## Building from source
+```bash
+git clone https://github.com/nim-lang/Nim
+cd Nim
+sh build_all.sh
+```
+The last command can be re-run after pulling new commits.
+Note that the `csources` repo used was changed to `csources_v1`, the new setup is designed to be forward and backward compatible.
+
+## Building from a CI setup
+We now have bash APIs to (re-)build Nim from source which hide implementation details, for example:
+`. ci/funs.sh && nimBuildCsourcesIfNeeded`.
+This can be useful for CI when alternatives (using nightly builds or a Docker image) are not suitable;
+in fact all the existing CI pipelines have been refactored to use this, see
+[#17815](https://github.com/nim-lang/Nim/pull/17815).
+
+
+# Contributors to Nim 1.6
+
+Many thanks to our recurring and new [contributors](https://github.com/nim-lang/Nim/graphs/contributors?from=2020-10-16&to=2021-09-25&type=c),
+Nim is a community driven collaborative effort that welcomes all contributions, big or small.
+
+
+# Backward compatibility and preview flags
```

Comment:
Is there a list with all such preview/legacy flags? If so, it should be linked here, otherwise, we should probably make one.

Reply:
I've finally added this, see text below "Here's the list of these flags"